docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -509,7 +560,9 @@ and age groups.
 Because we are not modeling individual causes of anemia (and their associated 
 mortality), we assume that all simulants have the same mortality rate regardless 
 of their hemoglobin value, when in reality, those with lower hemoglobin values 
-will have higher mortality rates than those with higher hemoglobin values.
+will have higher mortality rates than those with higher hemoglobin values. 
+Notably, deaths due to causes with iron responsive anemia sequelae account for 
+approximately 1.1% of deaths in the first five years of life globally. 
 
 Because hemoglobin concentrations are not directly modeled among the early and 
 late neonatal age groups in GBD, the prevalence of mild, moderate, and severe 
```

Comment:
lines 576-580, you have:

The modeling strategy both as conducted by the GBD modelers and as described in this document assume a constant shape and standard deviation in the hemoglobin distribution throughout the modeling process. This is a limitation of our modeling strategy in that we assume the distribution before a shift is applied maintains the same shift after a shift due to the intervention is applied.

is this 'the same *shape* after a shift is applies?

i still don't understand the implication of this, 😂 so the suggestion might be in error.

Reply:
Yes!! Thank you for catching this :)

Reply:
The implication is that the shift applies uniformly to all individuals regardless of their position in the hemoglobin distribution (imagine just pushing a normal distribution to the right, but its shape stays the same). However, we might actually expect the left end of the distribution to get pushed farther than the right end of the distribution in reality, so the shape would change from a normal distribution to a more lop-sided distribution if this were to happen (we assume it doesn't). I should add the the shape is only maintained in our simulation when stratified by iron responsiveness (because non-iron-responsive individuals will not get pushed at all).